.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
replace `.js` file format to `.eslintrc` or `.eslint.json` first one is preferable (not sure that preter will work with such)

Reply:
I've attempted to make the change. Does the latest commit include what you wanted?